lib/csv_import.rb

```diff
@@ -7,6 +7,10 @@ def csv_import(filepath)
     string = File.read(filepath)
     require 'csv'
 
+    # sanitize input
+    string = string.encode('UTF-8', 'binary',
+                           invalid: :replace, undef: :replace, replace: '')
```

Comment:
I used dummy CSVs for testing, and it didn't come up, so I think its fine. This was the solution presented in [this article](https://robots.thoughtbot.com/fight-back-utf-8-invalid-byte-sequences)